examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,418 @@
+      "    barrier;\n",
+      "    shift_phase(q10_q113_xy_frame, 1038.2659296624104);\n",
+      "    play(q10_q113_xy_frame, q10_q113_xy_sqrtiSWAP);\n",
+      "    shift_phase(q10_q113_xy_frame, -1038.2659296624104);\n",
+      "    shift_phase(q10_q113_xy_frame, 1040.74823388521 - 0.5*theta);\n",
+      "    play(q10_q113_xy_frame, q10_q113_xy_sqrtiSWAP);\n",
+      "    shift_phase(q10_q113_xy_frame, 0.5*theta - 1040.74823388521);\n",
+      "    delay[208.0ns] q10_rf_frame;\n",
+      "    shift_phase(q10_rf_frame, 5.359136323080053);\n",
+      "    delay[208.0ns] q113_rf_frame;\n",
+      "    shift_phase(q113_rf_frame, 5.747787462593388);\n",
+      "    shift_phase(q113_rf_frame, 0.5*theta);\n",
+      "    shift_phase(q10_rf_frame, -0.5*theta);\n",
+      "    barrier;\n",
+      "    shift_phase(q10_q11_xy_frame, 2.6795681615400264);\n",
+      "    shift_phase(q10_q17_xy_frame, 2.6795681615400264);\n",
+      "    shift_phase(q10_q113_xy_frame, 2.6795681615400264);\n",
+      "    shift_phase(q112_q113_xy_frame, -2.873893731296694);\n",
+      "    shift_phase(q113_q114_xy_frame, -2.873893731296694);\n",
+      "    shift_phase(q10_q113_xy_frame, -2.873893731296694);\n",
+      "    shift_phase(q112_q113_xy_frame, -0.25*theta);\n",
+      "    shift_phase(q113_q114_xy_frame, -0.25*theta);\n",
+      "    shift_phase(q10_q113_xy_frame, -0.25*theta);\n",
+      "    shift_phase(q10_q11_xy_frame, -0.25*theta);\n",
+      "    shift_phase(q10_q17_xy_frame, -0.25*theta);\n",
+      "    shift_phase(q10_q113_xy_frame, -0.25*theta);\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "xy_theta = XY(FreeParameter(\"theta\"))\n",
+    "rx_pi_2 = Rx(math.pi/2)\n",
+    "\n",
+    "a=10\n",
+    "b=113\n",
```

Comment:
L179 should use these. Can we rename these vars for when they appear later on for better context of the variable use